vision/google/cloud/vision/_gax.py

```diff
@@ -30,6 +32,27 @@ def __init__(self, client=None):
         self._client = client
         self._api = image_annotator_client.ImageAnnotatorClient()
 
+    def annotate(self, image, features):
+        """Annotate images through GAX.
+
+        :type image: :class:`~google.cloud.vision.image.Image`
+        :param image: Instance of ``Image``.
+
+        :type features: list
+        :param features: List of :class:`~google.cloud.vision.feature.Feature`.
+
+        :rtype: :class:`~google.cloud.vision.annotations.Annotations`
+        :returns: Instance of ``Annotations`` with results.
+        """
+        gapic_features = [_to_gapic_feature(feature) for feature in features]
+        gapic_image = _to_gapic_image(image)
+        request = image_annotator_pb2.AnnotateImageRequest(
+            image=gapic_image, features=gapic_features)
+        requests = [request]
+        api = self._api
+        responses = api.batch_annotate_images(requests)
+        return Annotations.from_pb(responses.responses[0])
```

Comment:
My typical complaint applies here: this assumes `len(responses.responses) == 1` but doesn't check it

Also, the access `responses.responses` indicates that `responses` isn't named correctly? What type does `api.batch_annotate_images` return?